06-best-practices/code/README.md

```diff
@@ -58,8 +76,88 @@ pylint --recursive=y .
 pytest tests/
 ```
 
-To prepare the project, run 
 
-```bash
-make setup
-```
\ No newline at end of file
+### Section 5: IaC
+w/ Terraform
+
+### Project infrastructure modules:
+* Amazon Web Service (AWS):
+    * S3 Bucket: artifacts
+    * Lambda: Serving API
+    * Kinesis: Streams (Producer & Consumer)
+
+### Setup
+
+1. If you've already created an AWS account, head to the IAM section, generate your secret-key, and download it locally. 
+[Instructions](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-prereqs.html)
+2. Download [aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) as a tool to use on your terminal
+3. Check installation
+  ```bash
+    $ which aws
+    /usr/local/bin/aws 
```

Comment:
what if it's a different location?

Reply:
also in the previous videos I use aws cli v1. Is it important to use v2? I also wouldn't include it here - I'd expect the students to have done it by now because we already used AWS previously